src/tests/ftest/util/dmg_utils.py

```diff
@@ -393,12 +409,35 @@ def pool_create(self, scm_size, uid=None, gid=None, nvme_size=None,
 
         # Extract the new pool UUID and SVC list from the command output
         data = {}
-        match = re.findall(
-            r"UUID:\s+([A-Za-z0-9-]+),\s+Service replicas:\s+([A-Za-z0-9-]+)",
-            self.result.stdout)
-        if match:
-            data["uuid"] = match[0][0]
-            data["svc"] = match[0][1]
+        if self.json.value:
+            # Sample json output.
+            # "response": {
+            #     "UUID": "ebac9285-61ec-4d2e-aa2d-4d0f7dd6b7d6",
+            #     "Svcreps": [
+            #     0
+            #     ]
+            # },
+            # "error": null,
+            # "status": 0
+            output = json.loads(self.result.stdout)
+            data["uuid"] = output["response"]["UUID"]
+            svc_list = output["response"]["Svcreps"]
+            svc_comma_separated = ""
+            for svc in svc_list:
+                if svc_comma_separated == "":
+                    svc_comma_separated = str(svc)
+                else:
+                    svc_comma_separated += ",{}".format(svc)
+            data["svc"] = svc_comma_separated
```

Comment:
Wouldn't this work?
```suggestion
            data["svc"] = ",".join(output["response"]["Svcreps"])
```

Reply:
if the items in `output["response"]["Svcreps"]` are not strings, you could always do this:
```
data["svc"] = ",".join([str(svc) for svc in output["response"]["Svcreps"]])
```

Reply:
Done. Thanks for suggestion.